src/cr/cube/matrix/assembler.py

```diff
@@ -1139,6 +1147,26 @@ def _subtotal_values(self):
         return measure_subtotal_rows[:, self._column_idx]
 
 
+class _SortRowsByDerivedColumnHelper(_SortRowsByBaseColumnHelper):
+    """Orders elements by the values of an opposing derived element vector.
+
+    This would be like "order rows in descending order by value of 'Coke' or 'Diet Coke'
+    array column. An opposing-element ordering is only available on a matrix, because only
+    a matrix dimension has an opposing dimension.
```

Comment:
Maybe find a place to work in that these only arise on subvariables dimensions, so far only MR but theoretically also possibly NUMA. The entire "inserted" subvariable has to be computed, like a value for every row, and that's why it needs to happen up on `zz9`.